### DISCUSSION TITLE
M2M 2.0 - adószámla lekérdezés

### DISCUSSION THREAD

**Discussion by kdenes6500** [nav-gov-hu/M2M#45] [Ötletek]
Ha az adószámla lekérdezhető lesz az M2M-en keresztül, akkor olyan megoldásra lesz szükségünk, mint az ügyfélportálon egy-egy adózó esetén lekérhető XML formátumú fájl. Ahhoz, hogy az ügyfelek számára tetszőleges kimutatásokat, értesítéseket tudjunk küldeni, részletes adatokra (adónem, dátum, összeg stb.) van szükség.
Fontos, hogy a lekérdezéskor egy jövőbeli dátum is megadható legyen. (Pl. tao-előírásokat is figyelembe akarom venni már a hónap elején. 2024.12.11-én tudjak 2024.12.20-ára vonatkozó lekérdezést indítani.)

**Comment by nanassy72** [comment]
A terveink között szerepel egy adószámla lekérdezést biztosító interfész megvalósítása az ÜPO-éhoz hasonló funkcionalitással, amivel várhatóan a jövőbeli kötelezettségek is lekérdezhetők lesznek.
